### ISSUE TITLE
Actively maintained

### ISSUE THREAD

**Issue by matukosan** [bryanmylee/svelte-headless-table#224]
hey guys, was wondering if this repo is still actively maintained. thinking about using it in a larger project, seen it used in shadcn-svelte, and want to make sure it will be around for some time 🤗

**Comment by fromaline** [comment]
@matukosan, I have the same exact question! @bryanmylee, are you still actively maintaining this project?

**Comment by bryanmylee** [comment]
Thanks for the check-in! Unfortunately, I've been unable to maintain this library actively as I no longer use Svelte, nor do I have any major plans to do so.

**Comment by fromaline** [comment]
@bryanmylee, that's really sad 😔
Anyway, I appreciate your work on this library — it's awesome!
Thanks a lot for the transparency!

**Comment by matukosan** [comment]
@bryanmylee agree, really sad to hear, since this was an awesome headless table implementation, better than many many others I've seen. really enjoyed playing around with it. good job! all the best for your future endeavors!

ps. pro'ly will fork it for my use cases 😁

**Comment by risalfajar** [comment]
Really sad to know this, I've used this lib for lots of clients project 😭

Could someone take over this lib?

**Comment by robots4life** [comment]
> Thanks for the check-in! Unfortunately, I've been unable to maintain this library actively as I no longer use Svelte, nor do I have any major plans to do so.

Thank you for Svelte Headless Table.

Out of curiosity, why do you no longer use Svelte and plan to not do so any more ?

**Comment by christophsturm** [comment]
what about mentioning this in the readme file, and maybe even archiving the repo?

**Comment by bryanmylee** [comment]
@robots4life @christophsturm @risalfajar I've just used Svelte less and less as I started working full-time, and haven't kept up to date with the Svelte ecosystem.

I'd be happy to add a notice on the README asking for new maintainers, and if you have any alternatives that you've had a great experience with, I'd gladly include it there too!

**Comment by jaysin586** [comment]
@bryanmylee How would you suggest we move forward? Do you have someone you would like to move this over to? How can we be of help in this process. I currently love the toolset and use it all over and want to make sure it continues on. 

@huntabyte have you seen this?

**Comment by justindomingue** [comment]
seems like shadcn-svelte switched to tanstack table https://next.shadcn-svelte.com/docs/components/data-table
